### ISSUE TITLE
Consider running the extension in Chrome's side panel

### ISSUE THREAD

**Issue by davidyuk** [superhero-com/superhero-wallet#2083]
https://developer.chrome.com/docs/extensions/reference/sidePanel/

The obvious advantages is that user can interact with webpage and extension at the same time. Also, confirmation popups can be showed in this side panel without opening a new window.

**Comment by peronczyk** [comment]
Would love to see this implemented. I think this does not require much work.

**Comment by peronczyk** [comment]
I implemented some changes in the app so user is able to run the extension both as a popup and the side panel (chrome / ff).

![Image](https://github.com/user-attachments/assets/813f683f-1304-4179-8c3a-1d42b547ba1d)

![Image](https://github.com/user-attachments/assets/65bd810a-1b28-40f4-ab89-944bdba554eb)


I was not able to establish any JS logic that would check if user opened the extension as the popup or side panel so for now I was not able to open confirmation modals within the side panel. I'll create a separate task for this.

**Comment by Liubov-crypto** [comment]
LGTM. 
Haven't tested apps.

**Comment by Liubov-crypto** [comment]
1. I've tested android app with Superhero DEX and it works as expected: 



https://github.com/user-attachments/assets/cecd5fb6-8e38-48d2-9d3a-f2e2e8f86aed



2. I tried to swap some ETH to WETH on Uniswap and after I click on swap button it doesn't open SH wallet confirmation popup or redirect me to SH wallet, I have to open wallet manually. Idk if it's expected behaviour: 



https://github.com/user-attachments/assets/41663bcb-0473-4f3f-9ad5-42333a5c21f6



3. Buttons look different size, but maybe it's phone problem. I'm using Samsung Galaxy S9 with small screen. 



![Image](https://github.com/user-attachments/assets/7112736a-eed9-44ca-a5fc-22e276ed33ff)
